app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -168,6 +171,9 @@ def clear_inapplicable_derived_values!
       self.wchair = nil
       self.location_id = nil
     end
+    if unittype_gn_changed? && unittype_gn_was == 2 && form.start_year_after_2024?
+      self.beds = nil
+    end
```

Comment:
Would this make more sense with the reset fields methods? Like `clear_inapplicable_derived_values!`

Reply:
This _is_ in `clear_inapplicable_derived_values`... Am I missing something?

Reply:
Oh, haha, I thought it was in `set_derived_fields!` together with the change above 🤦 My other comment now doesn't make sense either, ignore me